### PR TITLE
Update versioning and specify the command line for each case

### PR DESCRIPTION
**Pull request by marcoscv-work**
Add the versioning for las 7 version:  @^7.x.x
And specify the command line for each case, I think in this case is more clear

### CODE REVIEW

**Comment by mwilliams2014** [comment]
Thanks @marcoscv-work ! I'll merge this in.

File: develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-themes-generator.markdown [hunk 1]
```diff
@@ -76,19 +76,16 @@ You've completed installing the Liferay Theme Generator's dependencies.
...
-**Note:** Version 7.2.0 is the latest 7.x.x Theme Generator version at the time 
-of this writing. Always use the latest 7.x.x version to build with the most 
-recent fixes and updates.
-
-$$$
+To create themes for **6.2** and **7.0**:
 
+    npm install -g generator-liferay-theme@^7.x.x
+    
+To create themes for **7.0** / **7.1** / later versions of @product@:
```
Comment (mwilliams2014) [inline develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-themes-generator.markdown]: @marcoscv-work shouldn't this be 7.1 and later versions of the product? Version 7.x.x is for 7.0 right?

Comment (marcoscv-work) [reply develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-themes-generator.markdown]: Oh yes, it is a little bit strange but both versions can create 7.0 themes, this is 8.x.x:

image

and this: 7.x.x

image

Comment (mwilliams2014) [reply develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-themes-generator.markdown]: Ah, ok. That is a bit odd. Alright. Thanks!

**Comment by mwilliams2014** [comment]
Sent https://github.com/sez11a/liferay-docs/pull/3733 to 7.0.x branch and https://github.com/liferay/liferay-docs/pull/465 to 7.1

**Pull request closed without merging**